### DISCUSSION TITLE
<Image /> component causes 404

### DISCUSSION THREAD

**Discussion by paufar** [vercel/next.js#24041] [Help] [answered]
Hi, 
I have a folder called images in my public directory and im using the following to try and access the images inside it: 
`<Image src="/images/logos/logo.png" alt="Logo" width={300} height={300} />`

This works on my local but when deploying my application, i keep getting 404 errors for all images that im loading using the Image component. 
When moving one image directly to the public folder and using the following to load it, it works: 

`<Image src="/logo.png" alt="Logo" width={300} height={300} />`

Why are subdirectories not working? Is there some sort of config that needs to be set up before deploying on vercel?

**Comment by paufar** [comment, marked as answer]
Solved: Nested folders need to be inside /static for them to work. 
i.e /public/images will not work but /public/static/images works.

**Comment by inezabonte** [comment]
It depends on where you deploy. If you deploy on netlify for example you will have to use image optimizers since the `next\image` component has some issues there. But when you deploy on Vercel the creator of Next.js then you don't need to change anything. The first case would have still worked.
